analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -51,7 +51,10 @@ class Sbt(
     repoConfig: RepositoryConfiguration
 ) : PackageManager(name, analysisRoot, analyzerConfig, repoConfig), CommandLineTool {
     companion object {
-        private val VERSION_REGEX = Regex("\\[info]\\s+(\\d+\\.\\d+\\.[^\\s]+)")
+        // See https://github.com/sbt/sbt/blob/v1.5.1/launcher-package/integration-test/src/test/scala/RunnerTest.scala#L9.
+        private const val SBT_VERSION_PATTERN = "\\d(\\.\\d+){2}(-\\w+)?"
```

Comment:
Would it make sense to use REGEX as suffix for consistency with below variables?